PLOS/rhino
# ENG-122

https://jira.plos.org/jira/browse/ENG-122

This is pretty much the same as https://github.com/PLOS/wombat/pull/1635, but for rhino.
I had to bump the spring version, but all the tests pass and the code works, so this seems fine.

## To test:

1. Get the API key from https://github.com/PLOS/wombat/pull/1635
2. Start up rhino:
```
$ BUGSNAG_RELEASE_STAGE=development \
BUGSNAG_API_KEY=XXXX \
DATABASE_URL="jdbc:mysql://localhost:3306/ambra?user=egh&password=password" \
CONTENT_REPO_URL=http://dev-main.journals-contentrepo.service.consul:8002/v1/ \
THESAURUS=plosthes.2020-1 \
CORPUS_BUCKET=mogilefs-prod-repo \
TAXONOMY_URL=https://plos.accessinn.com:9138b/servlet/dh mvn clean package cargo:run
```

3. Generate an error, e.g. `curl http://localhost:8082/v2/config?type=buildxx`
4. Verify that error shows up in bugsnag.
